### ISSUE TITLE
Custom metrics - LabelNames property breaks registration of metric

### ISSUE THREAD

**Issue by petef19** [SkeLLLa/fastify-metrics#40]
Hi,

testing out your great repo. There is a bug in regards to registering custom metrics via the included prom-client:

Once the `LabelNames` property is being used (to declare labels for a custom metric), the metric registration fails. This seems to work just fine in the `prom-client` package, here's [an example regarding `LabelNames` taken from their docs](https://github.com/siimon/prom-client#labels) adjusted for fastify-metrics

```
// (1) this works
const res_gauge_1 = new fastify.metrics.client.Gauge({
    name: 'test_gauge_1',
    help: 'my test gauge 1',
});
// (2) this does NOT work
const res_gauge_2 = new fastify.metrics.client.Gauge({
    name: 'test_gauge_2',
    help: 'my test gauge 2',
    labelNames: ['test'],     // <--- breaks registration
});
```

See link above that this should work.

stack:
fastify-metrics: 7.4.0 (latest)
Node: 16.5.0
OS; Win 10 x64

Thanks.

**Comment by SkeLLLa** [comment]
Could you specify what exactly "fails" and how? Does it throw an error?

**Comment by petef19** [comment]
> Could you specify what exactly "fails" and how? Does it throw an error?

I just tested this in a different way, and it does work. The reason why I did not see the metric in the initial tests was, that when using labels, no initial value is set automatically (contrary to not using labels, where a default value of 0 is set), hence the metric did not show in Prometheus at all... I was able to see the metric though when directly looking at the metrics end point.

Sorry for the false alarm.

Maybe delete this thread to not confuse others.

Thanks !
